model/patch_lifecycle.go

```diff
@@ -625,6 +630,36 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	return patchVersion, nil
 }
 
+// getFullPatchParams will retrieve a merged list of parameters defined on the patch alias (if any)
+// with the parameters that were explicitly user-specified, with the latter taking precedence.
+func getFullPatchParams(p *patch.Patch) ([]patch.Parameter, error) {
+	paramsMap := map[string]string{}
+	if p.Alias != "" && IsPatchAlias(p.Alias) {
```

Comment:
can we just return p.Parameters in this case rather than re-iterating through at 650?

Reply:
I suppose it is true that we can skip re-iterating and just return `p.Parameters` at L650 if this boolean evaluates false. I've added an else block to check for this.

Reply:
I would rather we just did `if p.Alias == "" || !IsPatchAlias(p.Alias) { return p.Parameters, nil } ` at the top, since then we can have the rest of the work not in an if statement, which is cleaner to read.